docs/ops/sequence/CTCGreedyDecoder_1.md

```diff
@@ -24,20 +21,23 @@ Sequences in the batch can have different length. The lengths of sequences are c
 * *ctc_merge_repeated*
 
   * **Description**: *ctc_merge_repeated* is a flag for merging repeated labels during the CTC calculation.
-  * **Range of values**: true or false 
+  * **Range of values**: `true` or `false`
   * **Type**: `boolean`
-  * **Default value**: true 
+  * **Default value**: `true`
   * **Required**: *no*
 
 **Inputs**
 
-* **1**: `data` - Input tensor with a batch of sequences. Type of elements is any supported floating point type. Shape of the tensor is `[T, N, C]`, where `T` is the maximum sequence length, `N` is the batch size and `C` is the number of classes. Required.
+* **1**: `data` - input tensor with batch of sequences of type `T_F` and shape `[T, N, C]`, where `T` is the maximum sequence length, `N` is the batch size and `C` is the number of classes. **Required.**
 
-* **2**: `sequence_mask` - 2D input floating point tensor with sequence masks for each sequence in the batch. Populated with values 0 and 1. Shape of this input is `[T, N]`. Required.
+* **2**: `sequence_mask` - input tensor with sequence masks for each sequence in the batch of type `T_F` populated with values `0` and `1` and shape `[T, N]`. **Required.**
 
 **Output**
 
-* **1**: Output tensor with shape `[N, T, 1, 1]` and integer elements containing final sequence class indices. A final sequence can be shorter that the size `T` of the tensor, all elements that do not code sequence classes are filled with -1. Type of elements is floating point, but all values are integers.
+* **1**: Output tensor of type `T_F` and shape `[N, T, 1, 1]` which is filled with integer elements containing final sequence class indices. A final sequence can be shorter that the size `T` of the tensor, all elements that do not code sequence classes are filled with -1.
```

Comment:
Should be `-1` wrapped with back ticks?

Reply:
Added backticks in this [commit](https://github.com/openvinotoolkit/openvino/pull/5885/commits/b9e52dbd847023c04712d4f43246705f15ab017a).